dist/bundle.js

```diff
@@ -400,41 +400,48 @@ module.exports = function (cssWithMappingToString) {
 
 __webpack_require__.r(__webpack_exports__);
 /* harmony export */ __webpack_require__.d(__webpack_exports__, {
-/* harmony export */   "default": () => (__WEBPACK_DEFAULT_EXPORT__)
+/* harmony export */   "createGame": () => (/* binding */ createGame),
+/* harmony export */   "fetchGameData": () => (/* binding */ fetchGameData),
+/* harmony export */   "addScore": () => (/* binding */ addScore)
 /* harmony export */ });
+const baseUrl = 'https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/';
+
+let globalGameId;
 
-let scores = [
-  {
-    Name: 'Chike',
-    score: 71
-  },
-  {
-    Name: 'Nelsino',
-    score: 83
-  },
-  {
-    Name: 'Jose',
-    score: 78
-  },
-  {
-    Name: 'Dante',
-    score: 80
-  },
-  {
-    Name: 'Hamza',
-    score: 97
-  },
-  {
-    Name: 'Agbo',
-    score: 76
-  },
-  {
-    Name: 'Enio',
-    score: 98
-  },
-]
-
-/* harmony default export */ const __WEBPACK_DEFAULT_EXPORT__ = (scores);
+const createGame = () => {
+  const game = { name: 'My game' };
+  fetch(baseUrl, {
+    method: 'POST',
+    headers: {
+      'Content-Type': 'application/json',
+    },
+    body: JSON.stringify(game),
+  }).then((res) => res.json()).then((data) => {
+    if (!globalGameId) {
+      const gameId = data.result.split(' ')[3];
+      globalGameId = gameId;
+    }
+  });
+};
+
+createGame();
```

Comment:
- [ ] Please use `async` and `await` with the `fetch` method to send and receive data.